lesson4/style.css

```diff
@@ -0,0 +1,343 @@
+* {
+	box-sizing: border-box;
+}
+/************header**********/
+.header {
+	background-image: url(img/header_background.png);
+}
+.header__centre {
+	width: 80%;
+	margin: 0 auto;
+	display: flex;
+	justify-content: space-between;
+	padding: 10px 0;
+}
+.header__menu {
+	display: flex;
+	list-style: none;
+}
+.header__link {
+	color: #fff;
+	text-decoration: none;
+	margin-left: 25px;
+}
+.header__link:hover {
+	color: #56a1ce;
+}
+/************preview*********************/
+.preview {
+	background-image: url(img/grad.png);
+	background-size: cover;
+}
+.preview__centre {
+	width: 80%;
+	margin: 0 auto;
+	display: flex;
+	justify-content: flex-end;
+	position: relative;
+	padding-bottom: 25px;
+}
+.preview__info {
+	color: #fff;
+	width: 30%;
+	text-align: justify;
+	margin-top: 65px;
+	display: flex;
+	flex-direction: column;
+}
+.preview__title {
+	font-size: 20px;
+	text-transform: capitalize;
+	font-weight: bold;
+}
+.preview__text {
+	margin-top: 20px;
+	line-height: 22px;
+}
+.preview__button {
+	color: #fff;
+	font-size: 25px;
+	background-color: #56a1ce;
+	width: 190px;
+	height: 55px;
+	border: none;
+	border-radius: 3px;
+	margin: 60px auto 0;
+	cursor: pointer;
+}
+.page1 {
+	position: absolute;
+	top: 10%;
+	left: 17%;	
+}
+.page2 {
+	position: absolute;
+	top: 19%;
+	left: 0%;
+}
+.page3 {
+	position: absolute;
+	top: 30%;
+	left: 4%;
+	-webkit-box-reflect: below 2px -webkit-gradient(linear, left top, left bottom, from(transparent), color-stop(70%, transparent) , to(rgba(250, 250, 250, 0.1)));
+}
+/*********info***************/
+.info {
+	background-color: #fff;
+	padding-bottom: 60px;
+}
+.subscription {
+	width: 80%;
+	margin: 75px auto 0;
+	background-color: #d5d5d4;
+	color: #373737;
+	height: 70px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	padding: 10px;
+	position: relative;
+	z-index: 2; /* не знаю почему, но не работает */
+
+}
+.subscription:before, .subscription:after {
```

Comment:
1) you could use images. These elements is part of that block. Remove all z-index, remove using top and bottom, use only one. add height 1px to show shadow, and backgound: rgba(0,0,0,0,) to hide the block itself. You can't hide block and show shadow.